Simplenote/SimplenoteSimpleTextPrintFormatter.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  SimplenoteSimpleTextPrintFormatter.swift
+//  Simplenote
+//
+//  Created by Charlie Scheer on 1/10/20.
+//  Copyright © 2020 Automattic. All rights reserved.
+//
```

Comment:
Would it be possible to drop the copyright? we still need to remove that from several classes, but trying to keep new code without it.

(We do the same in WPiOS and Woo)